scripts/gen_profiles_layer.py

```diff
@@ -1251,39 +1251,44 @@ class JsonLoader {
     const auto &member = formats[format_name];
     for (const auto &name : member.getMemberNames()) {
         const auto &props = member[name];
-        if (name == "VkFormatProperties") {
+        if (name == "VkFormatProperties" || name == "VkFormatProperties3" || name == "VkFormatProperties3KHR") {
             for (const auto &feature : props["linearTilingFeatures"]) {
                 profile_properties.linearTilingFeatures |= StringToVkFormatFeatureFlags(feature.asString());
+                profile_properties_3.linearTilingFeatures |= StringToVkFormatFeatureFlags2(feature.asString());
```

Comment:
Why is this combined? And is it correct? You are passing the same `feature.asString()` string to different functions that expect different values. One has `VkFormatFeatureFlags` and the other `VkFormatFeatureFlags2`

Reply:
`VkFormatFeatureFlags2` maps to `VkFormatFeatureFlags` except there are more values.

Basically, the code will return nothing when it doesn't find the string.